pkg/udev/common.go

```diff
@@ -139,19 +176,21 @@ func (device *UdevDevice) getSize() (uint64, error) {
 // GetDevLinks returns syspath of a disk using syspath we can fell details
 // in diskInfo struct using udev probe
 func (device *UdevDevice) GetDevLinks() map[string][]string {
-	devLinkMap := make(map[string][]string)
-	byIdLink := make([]string, 0)
-	byPathLink := make([]string, 0)
+	devLinkUtil := newDevlinkUtil()
 	for _, link := range strings.Split(device.GetPropertyValue(UDEV_DEVLINKS), " ") {
 		parts := strings.Split(link, "/")
 		if util.Contains(parts, BY_ID_LINK) {
-			byIdLink = append(byIdLink, link)
```

Comment:
We should be ideally using array operations to append the required entry (after performing the filter) into the start of array.